components/browser/toolbar/src/main/res/values/ids.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!--
+  ~ This Source Code Form is subject to the terms of the Mozilla Public
+  ~  License, v. 2.0. If a copy of the MPL was not distributed with this
+  ~  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+  -->
+
+<resources>
+    <item name="toolbar_url_view" type="id"/>
```

Comment:
Since this change is going in a component, we probably want to use the same naming/scoping for our IDs?

Maybe `mozac_browser_toolbar_url_view` to avoid collisions?